### DISCUSSION TITLE
GC9A01 with Beetle-ESP32-C3???

### DISCUSSION THREAD

**Discussion by EmrysCES** [Bodmer/TFT_eSPI#3104] [Compatible displays and setup files]
Dear Bodmer,

A few weeks ago, I connected the GC9A01 display with the ESP32 chip. It worked like a charm but the chip is a bit big and clumsy, so i'm looking for a more compact alternative. I tried using the Beetle-ESP32-C3 but it just didnt want to happen. Can you please show me how or send me the post about connecting the seeed studio xiao esp32c3 with the GC9A01 display? Thanks a lot!

**Comment by Bodmer** [comment]
I have only tried the C3 with an ILI9341. I see no reason why it would not work with a GC9A01.

Setup Setup70c_ESP32_C3_ILI9341.h would be a good starting point for pin selection. You would need to change the display driver and may need to reduce the SPI frequency.
